src/components/Details/Details.tsx

```diff
@@ -48,8 +36,8 @@ export const Details: React.FC<MarketProps> = ({ id, markets, marketDetails }) =
 
   return (
     <section className='market-details'>
-      <h2>{nameMatch}</h2>
-      <p>Located at: {detail.street}, {detail.city} {detail.state}, {detail.zip}</p>
+      <h2>{selectedMarket?.name}</h2>
```

Comment:
This doesn't look like the Details component that I styled the other day. Missing that icon and the location formatting/amount of divs is allll kinds of different. Was this a merge issue?

Reply:
My main is up to date with upstream main